docs/user/03-70-delete-bindings-and-instances.md

```diff
@@ -40,4 +33,14 @@ To delete a service instance, run:
 ```bash
 kubectl delete serviceinstances.services.cloud.sap.com {SERVICE_INSTANCE_NAME}
 ```
-<!-- tabs:end -->
\ No newline at end of file
+<!-- tabs:end -->
+
+## Next Steps
+
+If you want to delete your Kyma cluster, you may have to first delete your service instances and bindings from it.
+
+You can't delete your Kyma cluster if any non-deleted service instances in it use the credentials from the SAP Service Manager resources created automatically, as described in [Preconfigured Credentials and Access](03-10-preconfigured-secret.md#credentials). In this case, the existing service instances block the cluster's deletion. Delete your service instances and bindings in Kyma dashboard before you attempt to delete the cluster from the SAP BTP cockpit.
+
+You can delete your Kyma cluster even if your service instances still exist, provided they all use credentials of SAP Service Manager service instances other than the one created automatically, as described in [Preconfigured Credentials and Access](03-10-preconfigured-secret.md#credentials). In this case, the non-deleted service instances do not block the cluster's deletion. 
```

Comment:
```suggestion
However, if the service instances use credentials of SAP Service Manager service instances other than the one created automatically, you can delete your Kyma cluster even if your service instances still exist (see [Preconfigured Credentials and Access](03-10-preconfigured-secret.md#credentials)). In this case, the non-deleted service instances do not block the cluster's deletion. 
```

Reply:
Same here.

Reply:
Skipping "all" and moving the link changes the message.